custom-namespaces.yaml

```diff
@@ -901,6 +913,10 @@ firefox_desktop:
       type: ping_explore
       views:
         base_view: default_browser
+    new_profiles:
```

Comment:
Maybe a name closer to the intended use-cases and that indicates that this is for Desktop? Personally I suggest building the explore directly in `spoke-default` after some experimentation/writing some queries